lib/versioned/runner.js

```diff
@@ -97,7 +98,19 @@ function installPackages(cb) {
 function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
-  spawn('node', [testFile], function testHandler(err) {
+  let args = [testFile]
+  if (process.env.packageType && process.env.packageType === 'module') {
+    // The default loader is in the agent root, so this uses process.cwd() instead of __directory.
+    const cwd = process.cwd().slice(0, process.cwd().indexOf('/test/'))
+    let loaderPath = path.resolve(`${cwd}/`, 'esm-loader.mjs')
+    if (process.env.loader) {
```

Comment:
maybe make this something that could be less of a conflict because technically this is opened source.  

`NR_LOADER`?

Reply:
Do we prefer the more-terse `NR_` or something like `NEW_RELIC_LOADER`? (Personally, I'd prefer terse.)

Reply:
sounds good to me